frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -1901,8 +1910,21 @@
             vm.method_description = vm.submissionMetaData.method_description;
             vm.project_url = vm.submissionMetaData.project_url;
             vm.publication_url = vm.submissionMetaData.publication_url;
-            vm.submissionId = submissionId;
-
+            vm.submissionId=submissionId;
+            if(vm.submissionMetaAttributes != null){
+            if(vm.submissionMetaAttributes.length > 0){
+            vm.submissionMetaAttributes[0].attributes.forEach(((attribute,index)=>{
```

Comment:
@burnerlee why are we just referring to `0th` index for submission meta attributes? Also please add a space after `,`

Reply:
oh I see, I've made a mistake here... fixed it in the latest commit